bin/seedUserStreaks.ts

```diff
@@ -0,0 +1,179 @@
+import fastq from 'fastq';
+import { differenceInDays } from 'date-fns';
+
+import '../src/config';
+import createOrGetConnection from '../src/db';
+import { User, UserStreak, View } from '../src/entity';
+import { DataSource, DeepPartial, Equal, IsNull, MoreThan, Not } from 'typeorm';
+import { DayOfWeek as Day } from '../src/types';
+
+const QUEUE_CONCURRENCY = 1;
+const BATCH_SIZE = 1000;
+
+interface UserPointer {
+  id?: string;
+  createdAt?: string;
+}
+
+const addNextBatch = async (
+  con: DataSource,
+  insertQueue: fastq.queueAsPromised<Partial<User>[], void>,
+  userPointer?: UserPointer,
+  limit: number = BATCH_SIZE,
+): Promise<UserPointer> => {
+  const qb = con
```

Comment:
Do we need to check that the user doesn't have a streak calculated already?
May be a good guard in case something breaks mid-run

Reply:
Currently there's no way to tell. Unless we want to store the ids of processed users in some intermediary table/redis. 

Even if there is a row in the `user_streak` table for the user, we don't know if it's correct numbers or not. Because since we released the newView worker changes, these numbers are incremented all the time, just not from initial values.

Reply:
I see. So let's just print the pointer after every iteration so we can start from that point